lang/cs/Org.Apache.REEF.Network/CommonStreamingCodecs/DoubleArrayStreamingCodec.cs

```diff
@@ -0,0 +1,100 @@
+﻿/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Org.Apache.REEF.Network.StreamingCodec;
+using Org.Apache.REEF.Tang.Annotations;
+using Org.Apache.REEF.Wake.Remote;
+
+namespace Org.Apache.REEF.Network.CommonStreamingCodecs
```

Comment:
Makes sense...I will put the folder as StreamingCodec'subfolder

On Mon, Jun 22, 2015 at 2:39 PM, Julia Wang <notifications@github.com>
wrote:

> In
> lang/cs/Org.Apache.REEF.Network/CommonStreamingCodecs/DoubleArrayStreamingCodec.cs
> <https://github.com/apache/incubator-reef/pull/236#discussion_r32986846>:
>
> > + * Unless required by applicable law or agreed to in writing,
> > + * software distributed under the License is distributed on an
> > + * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
> > + * KIND, either express or implied.  See the License for the
> > + * specific language governing permissions and limitations
> > + * under the License.
> > + */
> > +
> > +using System;
> > +using System.Threading;
> > +using System.Threading.Tasks;
> > +using Org.Apache.REEF.Network.StreamingCodec;
> > +using Org.Apache.REEF.Tang.Annotations;
> > +using Org.Apache.REEF.Wake.Remote;
> > +
> > +namespace Org.Apache.REEF.Network.CommonStreamingCodecs
>
> Why not put those classes into StreamingCodec folder?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/236/files#r32986846>.
>